site/content/_index.md

```diff
@@ -1,50 +1,34 @@
 ---
 title: Raise Green
 subtitle: 'Crowdfunding a healthy, just, and sustainable future.'
-image: /img/logo_no_text.png
+image: /img/solarpanels.jpg
 blurb:
   heading: About
   text: >-
-    Raise Green is a crowdfunding platform for impact investors to reduce air
-    pollution by deploying solar energy.
+    Raise Green is a crowdfunding platform for impact investors to acquire
+    equity in climate change solutions.
 intro:
+  heading: Our Vision
+  text: >-
+    Raise Green's vision is a healthy, just, and sustainable world where
+    communities experience a multitude of benefits from the democratized
+    ownership of renewable energy infrastructure.
+projects: []
+values:
   heading: Our Mission
   text: >-
-    Raise Green is the first crowdfunding platform focused on climate solution
-    projects with a social mission.
-
-
-    We combine the proven investment models of crowdsourcing and project finance to provide innovative pathways for
-    non-accredited investors to access sustainable investment opportunities in a
-    low-risk, low-cost, and impactful way. We aggregate small investments to
-    provide investors with equity into project finance companies. Initially,
-    this will take the form of solar projects, as the rate of return is very
-    predictable. However, in the future we plan to support the financing of an
-    array of climate solution projects.
+    Raise Green is committed to helping create a more healthy, just, and
+    sustainable world by:
 
 
-    Empower communities by funding projects from around the neighborhood and
-    around the county. Raise Green offers a tool for individuals to list the
-    projects they need in the community, then open the project up for
-    investment.
+    Confronting environmental challenges by implementing impactful projects,
```

Comment:
I feel like these should either be bullet points/subheadings or be slightly edited with the above sentence to read as a paragraph. We can accomplish either, might just need to tweak some of the template code to allow for bullet points in this section.

Reply:
Yeah bullets would work best, but I couldn't make them with in the CMS.